src/assetLoader.js

```diff
@@ -0,0 +1,35 @@
+import Manifest from './manifest';
+
+/**
+ * 
+ * @param {String | Array} path path to the string 
+ */
+export function getUrl(path) {
+    if (typeof path === "string") {
+        path = path.split('/');
+    }
+
+    // Check if path is empty
+    if (path === []) {
```

Comment:
This is not a valid comparison which will always be false.
`a = []; b = []; a === b (false)`
`a = []; b = []; a == b (false)`
The only way this works is:
`a = []; b = a; a === b (true)`
`a = []; b = a; a == b (true)`

Reply:
Yeah that's a mistake. Good catch